zuul.d/jobs-layout.yaml

```diff
@@ -4,6 +4,9 @@
     github-check:
       jobs:
         - noop
-        - adoption-standalone-to-crc-ceph
-        - adoption-standalone-to-crc-no-ceph
+        - adoption-standalone-to-crc-ceph: &required_projects
+            required-projects:
+              - name: openstack-k8s-operators/data-plane-adoption
+                override-checkout: main
+        - adoption-standalone-to-crc-no-ceph: *required_projects
```

Comment:
Wondering how is this change related...

Reply:
added by @ciecierski ^^ ?

Reply:
Yes, this was added by me. At some point of my debug of check job failures I changed order of Cleanup with Prelude roles imports and I didn't see this change in job log.
Then I reminded that we had similar problem in 18.0-fr2 branch https://github.com/openstack-k8s-operators/data-plane-adoption/pull/894/files

I will try to remove this overide-checkout and change order of Prelude and Cleanup to double-check it. There were a lot of moving parts in the change so I prefer to verify it.

Reply:
Here is test pull request: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/938